rdflib/namespace/__init__.py

```diff
@@ -722,6 +726,13 @@ def absolutize(self, uri: str, defrag: int = 1) -> URIRef:
         return URIRef(result)
 
 
+def _get_prefix_cc():
+    """Get the context from Prefix.cc."""
+    response = urlopen("https://prefix.cc/context.jsonld")
+    context = json.loads(response.read())
+    return context["@context"]
```

Comment:
One concern here is that there are some valid "@context" values [[ref](https://www.w3.org/TR/json-ld/#context-definitions)] that will not be handled correctly here, it may be that they don't appear in prefix.cc, but it may be good to confirm that somehow.

Reply:
yes, it's the case that there are no special values in this context dictionary based on the way that the context is constructed (https://github.com/cygri/prefix.cc/blob/cbc85c00e59e00cf4fee697374109fdd9027231a/templates/format/jsonld.php) and the strict requirements on prefixes (though right now I am having a hard time finding where it's documented that these have to be lowercase strings of alphanumeric characters length <= 10)